department-of-veterans-affairs/caseflow
# Update edit NOD permissions to just CoB

Resolves part of [CASEFLOW-1313](https://vajira.max.gov/browse/CASEFLOW-1313)

### Description
Updates permissions for editing NOD date from attorneys, judges, intake users, and Clerk of the Board to just Clerk of the Board.

### Acceptance Criteria

### Testing Plan
1. Log in as `COB_USER`
2. Ensure you can edit an AMA appeal (210315-47 for example)
3. Switch to a judge, attorney, and intake user and ensure you can see the change in timeline on the appeal that was edited, but cannot edit NOD date
